src/mc/nucleosome.f03

```diff
@@ -173,15 +290,187 @@ end subroutine setup_nucleosome_constants
 
 subroutine loadNucleosomePositions(wlc_nucleosomeWrap,wlc_basepairs)
     use precision, only: nan
+    ! sterics testing !
+    use GJKAlgorithm, only: GJK, sameShapeTest, noIntersectX, intersectX, tangentX, runtimeTest5, runtimeTest6, &
+                            noIntersectY, intersectY, tangentY, runtimeTest1, runtimeTest3, &
+                            noIntersectZ, intersectZ, tangentZ, runtimeTest2, runtimeTest4
+    use polydispersity, only: first_bead_of_chain, last_bead_of_chain
     implicit none
     integer, intent(out) :: wlc_nucleosomeWrap(WLC_P__NT)
-    integer, intent(out) :: wlc_basepairs(WLC_P__NT)
-
-    ! In the future you can set up code here to choose nucleosome spacing
-    wlc_nucleosomeWrap = 147
-    wlc_basepairs = WLC_P__LL
+    real(dp), intent(out) :: wlc_basepairs(WLC_P__NT)
+    real(dp), parameter :: L_in_bp = WLC_P__L/WLC_P__LENGTH_PER_BP
+    integer, parameter :: nNucs = nint((L_in_bp-WLC_P__LL)/(147+WLC_P__LL)) ! assuming all octasomes
+    real(dp) discretization, off_discretization, num_link_beads
+    real(dp), dimension(2,33) :: LL_dist
+    real(dp) cumlinker, linker
+    real(dp) urand(3)  ! random vector
+    integer iter, i, j
 
+    ! choose nucleosome spacing
```

Comment:
Could this be documented either inline or in docs.  It would be nice but not a deal-breaker for this pull request.

Reply:
ill try to add as much detail inline without going overboard and will definitely add more to outside docs.